app/models/notification.rb

```diff
@@ -12,36 +12,41 @@ class Notification < ApplicationRecord
   validates :status, presence: true
   validates :remind_on, presence: true
   validates :message, presence: true
+  validates :purpose, presence: true
 
   enum status: {
     pending: "pending",
     scheduled: "scheduled",
     sent: "sent",
     cancelled: "cancelled"
   }, _prefix: true
+  enum purpose: {
+    covid_medication_reminder: "covid_medication_reminder",
+    experimental_appointment_reminder: "experimental_appointment_reminder",
+    missed_visit_appointment_reminder: "missed_visit_appointment_reminder"
+  }, _suffix: true
```

Comment:
TBH, it's pretty clunky as either a prefix or suffix.

Reply:
Maybe just leave suffix and prefix off? These are long strings as they are, don't think they need the prefix/suffix.

Reply:
That's a good idea. Thanks. Hadn't occurred to me to just leave it off.